hw16_Class_Calculator/Task_1.ts

```diff
@@ -0,0 +1,49 @@
+//Task 1
+// Реализовать класс калькулятор, который принимает несколько методов: сложение, вычитание, умножение, деление
+// и принимает 2 параметра: 2 цифры.
+// Написать 10 положительных и отрицательных тестов по вызову данных функции.
+
+class Calculator {
+
+    constructor(){}
```

Comment:
If you are not setting something with constructor, you could omit it at all

Reply:
Removed.